prov/efa/src/rxr/rxr_ep.c

```diff
@@ -1432,12 +1431,80 @@ static inline void rxr_ep_check_peer_backoff_timer(struct rxr_ep *ep)
 	}
 }
 
-static inline void rxr_ep_poll_cq(struct rxr_ep *ep,
-				  struct fid_cq *cq,
-				  size_t cqe_to_process,
-				  bool is_shm_cq)
+/**
+ * @brief poll rdma-core cq and process the cq entry
+ *
+ * @param[in]	ep		end point
+ * @param[in]	cqe_to_process	max number of cq entry to poll and process
+ */
+static inline void rdm_ep_poll_ibv_cq(struct rxr_ep *ep,
+				      size_t cqe_to_process)
+{
+	struct ibv_wc ibv_wc;
+	struct efa_cq *efa_cq;
+	struct efa_av *efa_av;
+	struct efa_ep *efa_ep;
+	struct rdm_peer *peer;
+	struct rxr_pkt_entry *pkt_entry;
+	ssize_t ret;
+	int i;
+
+	VALGRIND_MAKE_MEM_DEFINED(&cq_entry, sizeof(struct fi_cq_data_entry));
```

Comment:
cq_entry is not defined ... pkt_entry? (to quote my compiler)

Reply:
What do you mean, it's defined on line 1438?

Reply:
I see that it is removed in commit ```prov/efa: rdm endpoint directly poll cq from ibv_cq``` sorry

Reply:
Thank you! Fixed in new revision.